### PR TITLE
Challenge brake

### PR DESCRIPTION
**Pull request by amackworth**
Nice and simple.

### CODE REVIEW

**Comment by celskeggs** [comment]
I'm not sure that "Emergency" is the right term here?

**Comment by amackworth** [comment]
We had a debate about it, that's what won. :stuck_out_tongue:

**Comment by celskeggs** [comment]
Why "Emergency", though?

**Comment by amackworth** [comment]
Ask Ben and Jake.

File: ZukoAzula2016/src/org/team1540/zukoazula/EmergencyBrake.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+public class EmergencyBrake {
+    private static final BooleanInput firstTrigger = ZukoAzula.controlBinding.addBoolean("Emergency Brake A");
+    private static final BooleanInput secondTrigger = ZukoAzula.controlBinding.addBoolean("Emergency Brake B");
+    private static final BooleanInput composedTriggers = FRC.inTeleopMode().and(firstTrigger).and(secondTrigger);
+
+    private static final FloatOutput servos = FRC.servo(0, -1, 1).combine(FRC.servo(1, -1, 1));
+
+    public static void setup() {
+        composedTriggers.toFloat(0, 1).send(servos);
```
Comment (celskeggs) [inline ZukoAzula2016/src/org/team1540/zukoazula/EmergencyBrake.java]: Shouldn't this be a toggle?

Comment (amackworth) [inline ZukoAzula2016/src/org/team1540/zukoazula/EmergencyBrake.java]: I was told both drivers should hold it down at once.

**Comment by amackworth** [comment]
Fiiiiiiiiiiiiiiine.

File: ZukoAzula2016/src/org/team1540/zukoazula/EmergencyBrake.java (hunk 1 above)
Comment (celskeggs) [inline ZukoAzula2016/src/org/team1540/zukoazula/EmergencyBrake.java]: Yes, but once they do that, it should remain stuck until the end of the match.

**Pull request merged**